src/ik_memetic.cpp

```diff
@@ -96,13 +96,7 @@ void MemeticIk::initPopulation(Robot const& robot,
     std::vector<double> const zero_grad(robot.variables.size(), 0.0);
     population_.resize(params_.population_size);
     for (size_t i = 0; i < params_.elite_size; ++i) {
-        auto genotype = initial_guess;
-        if (i > 0) {
-            for (size_t j_idx = 0; j_idx < robot.variables.size(); ++j_idx) {
-                auto const& var = robot.variables[j_idx];
-                genotype[j_idx] = rsl::uniform_real(var.clip_min, var.clip_max);
-            }
-        }
+        auto const genotype = (i == 0) ? initial_guess : robot.get_random_valid_configuration();
```

Comment:
@tylerjw C++ question: Does switching this implementation from modifying elements of a pre-existing vector to returning a brand new one and overwriting the variable affect performance significantly?

Wondering if it makes more sense for this new function to accept a mutable vector by reference instead.